### ISSUE TITLE
Wikipedia tabs seem to be missing images

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/OZtree#97]
E.g. on safari

http://www.onezoom.org/life.html/@Gracilinanus=608038?pop=ol_3606695#x411,y506,w1.0714

Which loads the iframe

http://www.onezoom.org/tree/wikipedia_OZpage?Q=194872&embed=1&leaf=1&name=Gracilinanus+dryas&wlang=en

In the console I get e.g.

```
Failed to load resource: http://upload.wikimedia.org/wikipedia/commons/e/e2/Wood_Sprite_Gracile_Mouse_Opossum_area.png
resource unavailable
```

Which is strange as this file is available online. Investigating

**Comment by hyanwong** [comment]
Works on chrome

**Comment by hyanwong** [comment]
For safari, we have to use https. This can probably be hacked in

**Comment by hyanwong** [comment]
So weirdly, it works permanently for ever more as soon as you go to wikipedia directly. I think this is something to do with the browser keeping track of where to send http requests. Possibly worth fixing by my change in 92a62be81b6219ed2697a741f4620e30df9e727a but maybe less urgent. I think I should consult with @lentinj first.

**Comment by hyanwong** [comment]
Fixed and merged into 3.6
